pkg/vpcmodel/grouping.go

```diff
@@ -97,29 +109,28 @@ type EndpointElem interface {
 	DrawioResourceIntf
 }
 
-type GroupedConnLine struct {
-	Src  EndpointElem
-	Dst  EndpointElem
-	Conn string
+type groupedConnLine struct {
+	src              EndpointElem
+	dst              EndpointElem
+	commonProperties *groupedCommonProperties // holds the common conn/diff properties
 }
 
-func (g *GroupedConnLine) String() string {
-	return g.Src.Name() + " => " + g.Dst.Name() + " : " + g.Conn
+func (g *groupedConnLine) String() string {
+	return g.src.Name() + " => " + g.dst.Name() + " : " + g.commonProperties.connStrKey
```

Comment:
`common.IsAllConnections(string)` is no longer needed and can be removed.

Reply:
done